lib/ReactViews/Workbench/Controls/DateTimeSelectorSection.js

```diff
@@ -64,6 +65,16 @@ const DateTimeSelectorSection = createReactClass({
         item.terria.currentViewer.notifyRepaintRequired();
     },
 
+    goToPrevDate(availableDates, currentIndex){
```

Comment:
Would probably go with Previous rather then the abbreviation...think thats the style guide but forget.